pipeline/frontend/yaml/constraint/constraint.go

```diff
@@ -137,9 +154,19 @@ func (c *Constraint) Match(metadata frontend.Metadata) bool {
 		match = match && c.Branch.Match(metadata.Curr.Commit.Branch)
 	}
 
+	if metadata.Curr.Event == frontend.EventCron {
+		// cron title is storend in message
+		match = match && c.Cron.Match(metadata.Curr.Commit.Message)
```

Comment:
Should we add the cron name as a new metadata property to be more explicit? This would allow us to expose it as env var and we could use commit message with the message of the last commit.